src/DataStructures/DataBoxTag.hpp

```diff
@@ -13,6 +13,7 @@
 #include "ErrorHandling/Assert.hpp"
 #include "Utilities/ConstantExpressions.hpp"
 #include "Utilities/ForceInline.hpp"
+#include "Utilities/NoSuchType.hpp"
```

Comment:
Why?

Reply:
I originally had the type be `NoSuchType` but figure `const void&` is a harder thing to abuse unintentionally. What's your opinion?